tests/stream/test_stream_smoke/0028_replay_stream.json

```diff
@@ -0,0 +1,618 @@
+{
+    "test_suite_name": "replay_stream",
+    "tag": "smoke",
+    "test_suite_config": {
+        "setup": {
```

Comment:
could you use format `yaml` for new smoke tests

Reply:
switch to `0029_replay_stream.yaml`, you can refer to `0028_emit_changelog.yaml`